config.toml

```diff
@@ -149,3 +149,9 @@ hrefTargetBlank = true
 lang = "en"
 languageName = "English"
 weight = 1
+
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
```

Comment:
do we need this?

Reply:
See https://github.com/inbo/tutorials/pull/324#issue-1800513065 and commit message of https://github.com/inbo/tutorials/commit/7db836f958760d0b79d3e04e38e477015f5115d9